### PR TITLE
Transfer vSwarm to the new vhive-serverless org

### PR DESCRIPTION
**Pull request by dhschall**
Moved now vSwarm dependencies.
Its a lot of dependencies. And not as easy as I thought at the beginning but maybe it will work.
The benchmarks for me except the once that uses XDT

### CODE REVIEW

**Comment by dhschall** [comment]
I needed to upgrade the builder as I already upgraded some of the mod files.
The mod files needed to be updated anyway because of new paths. I thought its better to then update the paths right away to 1.18. I hope this is fine.
Closes #299

**Comment by dhschall** [comment]
Linters fail for some of the benchmarks.
I think I should not try to fix it now. I created a separate issue as I think this should be fixed together with upgrading the golang version #300

File: README.md [hunk 1]
```diff
@@ -3,38 +3,42 @@
...
-This suite is a turnkey and fully tested solution meant to used in conjunction with 
+This suite is part of the [vHive Ecosystem](https://vhive-serverless.github.io/). Its a turnkey and fully
+tested solution meant to used in conjunction with
 [vHive](https://github.com/ease-lab/vhive), and is compatible with all technologies that it supports,
 namely, containers, Firecracker and gVisor microVMs. The majority of benchmarks support distributed
-tracing with [Zipkin](https://zipkin.io/) which traces both the infra components and the user 
+tracing with [Zipkin](https://zipkin.io/) which traces both the infra components and the user
 functions.
+Additionally, [several](./benchmarks/README.md#standalone-functions-benchmark-summary) of the benchmarks are compatible with
+[vSwarm-u](https://github.com/ease-lab/vSwarm-u), a framework that integrates the benchmarks with [gem5](https://www.gem5.org/),
```
Comment (ustiugov) [inline README.md:17]: Still old here

**Review by ustiugov — APPROVED** [review]
All good except one readme file. You can fix and merge

File: README.md (hunk 1 above)
Comment (dhschall) [reply README.md:17]: Good catch

Comment (dhschall) [reply README.md:17]: Ah remembered I did it because it was the first. But now everything is moved so now everything is clean

**Pull request merged**